docs/application/dotnet/samples/ui/overview.md

```diff
@@ -189,56 +189,56 @@ for TD:
 		<tr>
 			<td><img alt="" height="180" src="media/w46_animationsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) AnimationSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AnimationsSample" target="_blank"><strong>(W) AnimationSample (NUI)</strong></a></p>
```

Comment:
Global comment:
There are some inconsistencies in the names of the sample for Wearable and TV, and also inconsistencies with the titles of respective readme files.

For example,
For Wearable samples, for "(W) Rotary timer", the title of readme file is "RotaryTimer".

One more issue I have seen is that in many files the title and the UI names are also having inconsistency. 
For example,
![tv_text_sample](https://user-images.githubusercontent.com/36875286/55212824-2581bc80-51e9-11e9-961c-38dbf670708d.JPG)

I think this has been missed by the reviewers in the past. Apologies for the same.
Going forward let us fix these issues as well. Please correct me if I am wrong in my assessment regarding the same.

If this change is not valid for this PR, please ignore my comments.

Reply:
README page were made by each application developers
I will send an request email for updating README pages.